docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -145,6 +145,22 @@ You can enable TLS v1.0 by configuring the relevant `ssl.supported_protocols` se
 xpack.security.http.ssl.supported_protocols: [ "TLSv1.2", "TLSv1.1", "TLSv1" ]
 --------------------------------------------------
 
+[float]
+[[trial-explicit-security]]
+==== Security on Trial Licenses
+
+On trial licenses, `xpack.security.enabled` defaults to `false`.
+
+In prior versions, a trial license would automatically enable security if either
+
+* `xpack.security.transport.enabled` was `true`; _or_
```

Comment:
Needed, no.

Semicolons within lists, and semicolons before conjunctions are both acceptable, but a little bit outdated.
I only use them when (like this case) you have a multi-line list (bullet points, or numbers) and a conjunction that relates to the overall list. In this case the `;` makes clear that the _or_ is not part of the first point (it would be grammatically incorrect if it were, but may still confuse readers) but applies to the list itself.

Reply:
aha! They don't teach you that stuff at English school :) (or they did and I have forgotten) - Thanks for the clarification